components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -255,25 +253,23 @@ class AdminSidebar extends React.PureComponent<Props, State> {
                     return null;
                 }
 
-                if (sidebarItems.length) {
```

Comment:
is it safe to remove the guard?

Reply:
This change was based on your [previous comment](https://github.com/mattermost/mattermost-webapp/pull/10303#discussion_r874430328)